app/styles/modules/_m-orange-bar.scss

```diff
@@ -134,3 +134,22 @@ ul .menu li.dropdown {
 .dropbtn-left {
   justify-content: left;
 }
+
+.orange-bar-custom-map-study {
```

Comment:
I moved this to a orange-bar-custom-map-study specific file:

`modules/components/custom-map-study/toolbar`